Add FileNameSigner and register in SignerStore (#455)
repository-service-tuf/repository-service-tuf-worker: docs/source/guide/Docker_README.md

  Example: ``RSTUF_LOCAL_KEYVAULT_KEYS=/run/secrets/ONLINE_KEY_1:/run/secrets/ONLINE_KEY_2``

#### (Optional, *experimental*) `RSTUF_ONLINE_KEY_DIR`

> Directory path to online signing key. The setting is required, if the related
> public key metadata includes a field `x-rstuf-online-key-uri` with a value
> `fn:<file name>`. The full path is constructed by joining directory path
> and file name. The expected private key file format is unencrypted PKCS8/PEM.

Reviewer: nit: focus on the deployment without need deep info on the TUF metadata structure or Signer API

```suggestion
Directory path to online signing key. The setting is required if using provide the online key as file. The expected private key file format is unencrypted PKCS8/PEM.
```

Author: > nit: focus on the deployment without need deep info on the TUF metadata structure or Signer API

Makes sense. 

I think there is a stray "provide" in your suggestion. What about just
 
```suggestion
Directory path for online signing key file. Expected private format is unencrypted PKCS8/PEM.
```
